ui/src/App.tsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import Header from './Header';
+import Routes from './Routes';
+
+import firebase from './utils/firebase';
+
+const App: React.FC = () => {
+  const [currentUser, setCurrentUser] = React.useState<firebase.User | null>(null);
```

Comment:
Not sure if it makes sense to manage `currentUser` in the state of the main component, or if I should use a context or something else instead 🤔 .